From 569010f4499e746c165b56752199ece3fac2d1b9 Mon Sep 17 00:00:00 2001
From: "jonathan.de.cesco" <jonathan.de.cesco@intel.com>
Date: Mon, 13 Aug 2012 11:14:13 +0200
Subject: [PATCH 15/78] WLAN: calibrator add stderr outputs to aplogs

BZ: 43903

replaces fprintf(stderr,...) by LOGE

Change-Id: Iade6d4121173724606095c700c89c6801a0a615c
Orig-Change-Id: If00651b926a1ceda8e8b55585b4308dc39563b6f
Signed-off-by: Maher Khenissi <maherx.khenissi@intel.com>
Signed-off-by: jonathan.de.cesco <jonathan.de.cesco@intel.com>
Reviewed-on: http://android.intel.com:8080/61811
Reviewed-by: Regairaz, Julie <julie.regairaz@intel.com>
Reviewed-by: Lemetayer, Jean-marieX <jean-mariex.lemetayer@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Yin, Fengwei <fengwei.yin@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/77972
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 calibrator/Android.mk   |    2 ++
 calibrator/calibrator.c |    4 ++++
 calibrator/ini.c        |    4 ++++
 calibrator/misc_cmds.c  |    4 ++++
 calibrator/nvs.c        |    5 +++++
 calibrator/plt.c        |    4 ++++
 6 files changed, 23 insertions(+), 0 deletions(-)

diff --git a/calibrator/Android.mk b/calibrator/Android.mk
index 9eca74f..8b1054a 100644
--- a/calibrator/Android.mk
+++ b/calibrator/Android.mk
@@ -23,4 +23,6 @@ LOCAL_STATIC_LIBRARIES := libnl_2
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := calibrator
 
+LOCAL_SHARED_LIBRARIES:= libcutils
+
 include $(BUILD_EXECUTABLE)
diff --git a/calibrator/calibrator.c b/calibrator/calibrator.c
index 42ecd61..ea759ef 100644
--- a/calibrator/calibrator.c
+++ b/calibrator/calibrator.c
@@ -3,6 +3,7 @@
  *
  * See README and COPYING for more details.
  */
+#define LOG_TAG "Calibrator"
 
 #include <errno.h>
 #include <stdio.h>
@@ -13,6 +14,7 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <stdbool.h>
+#include <cutils/log.h>
 
 #include <netlink/genl/genl.h>
 #include <netlink/genl/family.h>
@@ -25,6 +27,8 @@
 #include "plt.h"
 #include "ini.h"
 
+#define fprintf(out,...) LOGE(__VA_ARGS__)
+
 char calibrator_version[] = "0.73";
 #ifndef CONFIG_LIBNL20
 /* libnl 2.0 compatibility code */
diff --git a/calibrator/ini.c b/calibrator/ini.c
index f3bb4eb..a41fb5b 100644
--- a/calibrator/ini.c
+++ b/calibrator/ini.c
@@ -5,6 +5,7 @@
  *
  * See README and COPYING for more details.
  */
+#define LOG_TAG "Calibrator"
 
 #include <sys/ioctl.h>
 #include <errno.h>
@@ -16,6 +17,7 @@
 #include <unistd.h>
 #include <stdbool.h>
 #include <limits.h>
+#include <cutils/log.h>
 
 #include <netlink/genl/genl.h>
 #include <netlink/genl/family.h>
@@ -30,6 +32,8 @@
 #include "ini.h"
 #include "nvs.h"
 
+#define fprintf(out,...) LOGE(__VA_ARGS__)
+
 static char *ini_get_line(char *s, int size, FILE *stream, int *line,
 				  char **_pos)
 {
diff --git a/calibrator/misc_cmds.c b/calibrator/misc_cmds.c
index a650eaf..a499213 100644
--- a/calibrator/misc_cmds.c
+++ b/calibrator/misc_cmds.c
@@ -1,3 +1,4 @@
+#define LOG_TAG "Calibrator"
 #include <stdbool.h>
 #include <errno.h>
 #include <net/if.h>
@@ -7,11 +8,14 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <sys/mman.h>
+#include <cutils/log.h>
 #include "calibrator.h"
 #include "plt.h"
 #include "ini.h"
 #include "nvs.h"
 
+#define fprintf(out,...) LOGE(__VA_ARGS__)
+
 SECTION(get);
 SECTION(set);
 
diff --git a/calibrator/nvs.c b/calibrator/nvs.c
index a2ee1c8..b55c3f5 100644
--- a/calibrator/nvs.c
+++ b/calibrator/nvs.c
@@ -3,6 +3,7 @@
  *
  * See README and COPYING for more details.
  */
+#define LOG_TAG "Calibrator"
 
 #include <sys/ioctl.h>
 #include <errno.h>
@@ -15,6 +16,7 @@
 #include <stdbool.h>
 #include <netinet/in.h>
 #include <time.h>
+#include <cutils/log.h>
 
 #include <netlink/netlink.h>
 #include <netlink/msg.h>
@@ -25,6 +27,9 @@
 #include "calibrator.h"
 #include "plt.h"
 #include "ini.h"
+
+#define fprintf(out,...) LOGE(__VA_ARGS__)
+
 /* 2048 - it should be enough for any chip, until... 22dec2010 */
 #define BUF_SIZE_4_NVS_FILE	2048
 
diff --git a/calibrator/plt.c b/calibrator/plt.c
index a6f5b42..704819b 100644
--- a/calibrator/plt.c
+++ b/calibrator/plt.c
@@ -3,6 +3,7 @@
  *
  * See README and COPYING for more details.
  */
+#define LOG_TAG "Calibrator"
 
 #include <sys/ioctl.h>
 #include <errno.h>
@@ -13,6 +14,7 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <stdbool.h>
+#include <cutils/log.h>
 
 #include <netlink/genl/genl.h>
 #include <netlink/genl/family.h>
@@ -30,6 +32,8 @@
 
 #define ZERO_MAC	"00:00:00:00:00:00"
 
+#define fprintf(out,...) LOGE(__VA_ARGS__)
+
 #ifndef SIOCETHTOOL
 #define SIOCETHTOOL     0x8946
 #endif
-- 
1.7.9

